src/components/MarketActions/index.tsx

```diff
@@ -75,15 +75,13 @@ function MarketActionsDesktop({ market, canRepayAndWithdraw }: MarketActionsProp
         </div>
       )}
 
-      <div className="rounded-[12px] bg-[#3B2870]">
```

Comment:
This is actually needed:

![Screenshot 2025-05-09 at 11 45 27](https://github.com/user-attachments/assets/fe0ba1e3-385a-4fe7-9123-d48f28427057)